docs/content/docs/dev/datastream/fault-tolerance/state.md

```diff
@@ -474,9 +474,10 @@ ttl_config = StateTtlConfig \
 For more fine-grained control over some special cleanup in background, you can configure it separately as described below.
 Currently, heap state backend relies on incremental cleanup and RocksDB backend uses compaction filter for background cleanup.
 
-##### Cleanup in full snapshot
+##### Cleanup in full scan snapshot
 
-Additionally, you can activate the cleanup at the moment of taking the full state snapshot which 
+Additionally, you can activate the cleanup at the moment of taking the full scan state snapshot (means the 
+canonical savepoint of all state-backends, or the checkpoint/native-savepoint of hashmap state-backend.), which 
```

Comment:
Hmm..I was thinking that RocksDB non-incremental checkpoints also support this TTL mode, don't they?

Reply:
After FLINK-28699, it was not true.

Reply:
Will RocksDB non-incremental checkpoints support `cleanupInRocksdbCompactFilter` After FLINK-28699？

Reply:
@fredia sure, cleanup with compaction filter is only related with RocksDB's background compaction and has no direct relationship with what kind of snapshot.